packages/cursorless-everywhere-talon-core/src/registerCommands.ts

```diff
@@ -0,0 +1,49 @@
+import { CURSORLESS_COMMAND_ID } from "@cursorless/common";
+import type { CommandApi } from "@cursorless/cursorless-engine";
+import type { TalonJsIDE } from "./ide/TalonJsIDE";
+import type { Talon } from "./types/talon.types";
+
+export function registerCommands(
+  talon: Talon,
+  ide: TalonJsIDE,
+  commandApi: CommandApi,
+) {
+  const ctx = talon.Context();
+  ctx.matches = "not app: vscode";
```

Comment:
We should do something more generic. For example this will now break cursorless neovim, which is already shipped

Reply:
We could eider keep adding applications here or we need a new tag.

Reply:
I don't feel strongly, but probably better to have something where we don't need to remember to update this every time we add a new application

Reply:
update from meet-up: have a tag, and then a README in this dir explaining how to activate it for `not vscode`